.github/workflows/ef-tests.yml

```diff
@@ -0,0 +1,78 @@
+name: EF Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions: read-all
+
+jobs:
+  ef-tests:
+    runs-on: ubuntu-latest
```

Comment:
how long does this take to run with a little machine such as this?

Reply:
yes for sure

Reply:
bumped to 32 cores, which is what ef-tests is using for its CI.